modules/observe/scorecards/proc-configuring-jira-scorecards-in-rhdh-instance.adoc

```diff
@@ -0,0 +1,177 @@
+
+. Add your Jira configuration (${JIRA_TOKEN}) to {configuring-book-link}#provisioning-your-custom-configuration[your {product-very-short} secrets] using the following key/value pairs:
+*** `JIRA_TOKEN`
+*** `JIRA_BASE_URL`
+
+. Install the Jira Scorecard plugin in your {product-very-short} `dynamic-plugins-config.yaml` using either a direct setup or a proxy setup.
+.. Use a direct setup: Add the following code to your {product-very-short} `dynamic-plugins-config.yaml`:
++
+[source,yaml]
+----
+plugins:
+  - package: oci://ghcr.io/redhat-developer/rhdh-plugin-export-overlays/red-hat-developer-hub-backstage-plugin-scorecard-backend-module-jira:pr_1499__0.1.0!red-hat-developer-hub-backstage-plugin-scorecard-backend-module-jira
+    disabled: false
+----
+... In your {product-very-short} `{my-app-config-file}` file, add the following direct setup settings:
++
+[source,yaml]
+----
+jira:
+  baseUrl: ${JIRA_BASE_URL}
+  token: ${JIRA_TOKEN}
+  product: _<jira_product>_
+----
+where:
+
+`baseUrl`:: The base URL of your Jira instance, configured under ${JIRA_BASE_URL} in your {product-very-short} secrets.
+`token`:: The Jira token (Base64 string for Cloud, PAT for Data Center), configured under ${JIRA_TOKEN} in your {product-very-short} secrets.
+`product`:: Enter the supported product: `cloud` or `datacenter`.
+
+.. Use a proxy setup: In your {product-very-short} `dynamic-plugins-config.yaml`, add the following code:
++
+[source,yaml]
+----
+plugins:
+  - package: oci://ghcr.io/redhat-developer/rhdh-plugin-export-overlays/red-hat-developer-hub-backstage-plugin-scorecard-backend-module-jira:pr_1499__0.1.0!red-hat-developer-hub-backstage-plugin-scorecard-backend-module-jira
+    disabled: false
+----
++
+[NOTE]
+====
+You must use the proxy setup to ensure configuration compatibility if the Roadie Jira Frontend Plugin is also in use.
+====
+
+... In your {product-very-short} `{my-app-config-file}` file, add the following proxy settings:
++
+[source,yaml]
+----
+proxy:
+  endpoints:
+    '/jira/api':
+      target: ${JIRA_BASE_URL}
+      headers:
+        Accept: 'application/json'
+        Content-Type: 'application/json'
+        X-Atlassian-Token: 'no-check'
+        Authorization: ${JIRA_TOKEN} # Must be configured in your environment
+jira:
+  proxyPath: /jira/api
+  product: cloud # Change to 'datacenter' if using Jira Datacenter
+----
+where:
+
+`target`:: The base URL of your Jira instance, configured under ${JIRA_BASE_URL} in your {product-very-short} secrets.
+`Authorization`:: The Jira token, configured under ${JIRA_TOKEN} in your {product-very-short} secrets. Set the token value as one of the following values:
+* For *Cloud*: `Basic YourCreatedCloudToken`
+* For *Data Center*: `Bearer YourJiraToken`
+
+. Annotate the component entity in the `catalog-info.yaml`.
+.. In the `catalog-info.yaml` file for the entity you want to configure, link the component to the Jira data source by adding the necessary annotations:
```

Comment:
```suggestion
..  Link the component to the Jira data source by adding the necessary annotations:
```
Minimalism as the context is already clear from the previous sentence. We can make it a bullet list.